orgs/beta_platformencryption.json

```diff
@@ -0,0 +1,17 @@
+{
+  "orgName": "NPSP - Beta Org - Platform Encryption",
+  "edition": "Developer",
+  "instance": "cs46",
```

Comment:
Is this intentionally a Winter '20 scratch org?

Reply:
Yes, bu tit doesn't need to stay that way. Winter '20 did not change the Metadata API behavior I wanted it to change.